itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosExternalTablesMetaDataOnly.java

```diff
@@ -639,9 +629,11 @@ public void testIncrementalDumpEmptyDumpDirectory() throws Throwable {
             .verifyResult(inc2Tuple.lastReplicationId);
   }
 
-  private void assertFalseExternalFileList(Path externalTableFileList)
-      throws IOException {
+  private void assertFalseExternalFileList(String dumpLocation)
+          throws IOException {
```

Comment:
I think it doesn't throw IOException

Reply:
getFileSystem() call throws IOException.